src/main/java/mafia/mafiatogether/common/config/WebSocketConfig.java

```diff
@@ -34,9 +39,18 @@ public void configureMessageBroker(MessageBrokerRegistry registry) {
         registry.enableSimpleBroker("/sub");
     }
 
+    @Override
+    public void addArgumentResolvers(final List<HandlerMethodArgumentResolver> argumentResolvers) {
+        argumentResolvers.add(new WebsocketPlayerArgumentResolver());
+    }
```

Comment:
```suggestion
    @Override
    public void addArgumentResolvers(final List<HandlerMethodArgumentResolver> argumentResolvers) {
        argumentResolvers.add(new PlayerArgumentResolver());
    }
```

이거는 안되나?

`WebSocketPlayerArgumentResolver`가 `PlayerArgumentResolver`랑 똑같이 구현되어있던데 다시 구현해야하는 이유가 있어?

는줄 알았는데 이름만 같은 다른 인터페이스를 상속받고있네;;;;

ㄷㄷ

심지어 구현하는 매서드 명도 똑같네, 인자만 다를뿐...

`PlayerArgumentResolver`가 두개 다 구현하게 하면 너무 그런가?
```java
public class PlayerArgumentResolver implements
        HandlerMethodArgumentResolver,
        org.springframework.messaging.handler.invocation.HandlerMethodArgumentResolver
```
구현체를 나눌지 아니면 합쳐서 하는 방법도 있긴한데... 에반가?

Reply:
오우 좋은데 
```java
public class PlayerArgumentResolver implements
        HandlerMethodArgumentResolver,
        org.springframework.messaging.handler.invocation.HandlerMethodArgumentResolver
```

이렇게 쓰면 한곳에서 관리할수있을거같아

Reply:
아 근데 난 잘 판단이 안서는게 어차피 같은 로직이라 한 클래스에서 저렇게 관리하는게 맞는거 같다고는 생각이 드는데
클래스 네임도 메서드 네임도 똑같아서 조금 그렇고

스프링에서 저걸 뭔가 인터페이스를 저렇게 구현한게 그 뭐냐 인터페이스 분리 원칙? 때문에 분리해서 구현한거라고 생각하면 또 따로 구현하는게 또 맞지 않을까라는 생각도 들어서 어떻게 생각하는가 물어본거...

이걸 팬딩해놓고 안보냈네 ㅋㅋㅋ

Reply:
인터페이스를 저렇게 분리한 것도 잘 분리해놓은 것 같아요.

사실 뭐 나눠서 구현하던, 아니면 달리가 추천해준 방식처럼 implements 두 개 받아서 구현하던 개발자 자기 마음인 것 같고, 그것도 나름 이를 잘 활용한 것이라고 생각되네요!

너무 좋은데요 달리 ?